src/client/containers/FavoritesPage/FavoritesPage.Container.js

```diff
@@ -1,10 +1,52 @@
-import React from 'react';
+import { useParams } from 'react-router-dom';
+import React, { useEffect } from 'react';
 import './FavoritesPage.Style.css';
+import { ProductDetails } from '../../components/ProductDetails/ProductDetails.component';
 
 const FavoritesPageContainer = () => {
+  const [favorites, setFavorites] = React.useState([]);
+
+  const { id } = useParams();
+  React.useEffect(() => {
```

Comment:
why don`t use custom hook created  -> `useFetch`

Reply:
Thank you, it is a good point.
I`ve changed, please review